build.gradle

```diff
@@ -4,11 +4,11 @@ plugins {
     id "java"
     id "cpp"
     id "com.diffplug.spotless" version "6.24.0"
-    id "edu.wpi.first.NativeUtils" version "2024.6.1" apply false
+    id "edu.wpi.first.NativeUtils" version "2025.4.0" apply false
     id "edu.wpi.first.wpilib.repositories.WPILibRepositoriesPlugin" version "2020.2"
-    id "edu.wpi.first.GradleRIO" version "2024.3.2"
+    id "edu.wpi.first.GradleRIO" version "2025.1.1-beta-1"
     id 'edu.wpi.first.WpilibTools' version '1.3.0'
-    id 'com.google.protobuf' version '0.9.4' apply false
+    id 'com.google.protobuf' version '0.9.3' apply false
```

Comment:
Is this still needed after nuking protobuf C++?

Reply:
I'm actually not sure. We use quickbuf for Java code gen. FAFO?

Reply:
OK I just checked, we do use this but since we are using quickbuf we should switch to something just using a python script as we override everything